#BE-3744 Docs Update - Team Activity Log (#486)
appcircleio/appcircle-docusaurus: docs/account/my-organization/team-activity-log.md

---
title: Team Activity Log
description: Appcircle will notify external services via webhooks when a certain event occurs. When the events you specified happen, we'll send a POST request in JSON format to the URLs you provide.
tags:
  [
    organizations,
    sub organizations,
    team activity log,
    organization membership,
    role management,
    activity log,
  ]

---

> Team management actions, such as Organization or Sub Organization creations, deletions and member additions, can be viewed at Team Activity Log section within the My Organization area.

Reviewer: ```suggestion
You can view team management actions such as creating, deleting, and adding members to Organizations or Sub Organizations in the Team Activity Log section within the My Organization area.
```

- Removing a member from an Organization / Sub-Organization
- A member leaving an Organization / Sub-Organization
- Updating a member's role within an Organization / Sub-Organization

:::caution

Only Organization / Sub-Organization Owners and users with Organization Management Role will have access to this area.

Information about other Organizations and their Sub-Organizations will not be accessible without the required level of clearance.

:::

:::info

Organization Owners can also observe the team activity actions of their Sub-Organizations.

:::

You can edit the required date range by clicking the time filter in the top right corner and selecting a time limit from various options. Alternatively, you can choose custom dates from the calendar.

<Screenshot url='https://cdn.appcircle.io/docs/assets/be-3744-teamactivity4.png' />

Team activity logs also include filters to help users perform more precise searches. By clicking the 'All' option next to Organizations, you can select a specific organization or sub-organization from the list, provided you have access to monitor their team activity.

<Screenshot url='https://cdn.appcircle.io/docs/assets/be-3744-teamactivity2.png' />

> Another search method is to search by 'Actions'. Simply click the Organization drop-down box and select 'Actions'. From there, you can choose a specific action to refine your search.

Reviewer: ```suggestion
Another method to search is by **Actions**. Simply click the Organization drop-down box, select **Actions**, and then choose a specific action to refine your search.
```
